internal/controller/utils.go

```diff
@@ -285,3 +287,19 @@ func SetDefaults_Deployment(obj *appsv1.Deployment) {
 		*obj.Spec.ProgressDeadlineSeconds = 600
 	}
 }
+
+func getQueryFilters(cr *olsv1alpha1.OLSConfig) []QueryFilters {
```

Comment:
this function seems a better fit for `ols_app_server_assets.go` as it is only for the application server's config.